packages/grafana-ui/src/components/index.ts

```diff
@@ -264,8 +264,9 @@ export { Avatar } from './UsersIndicator/Avatar';
 // Export this until we've figured out a good approach to inline form styles.
 export { InlineFormLabel } from './FormLabel/FormLabel';
 export { Divider } from './Divider/Divider';
-export { getDragStyles } from './DragHandle/DragHandle';
-export { Splitter } from './Splitter/Splitter';
+export { getDragStyles, type DragHandlePosition } from './DragHandle/DragHandle';
+export { useSplitter } from './Splitter/useSplitter';
+export { useSnappingSplitter } from '../../../../public/app/features/dashboard-scene/panel-edit/splitter/useSnappingSplitter';
```

Comment:
we shouldn't expose this from `@grafana/ui`

Reply:
@ashharrison90 good catch, missed that vscode added this when I moved it